File: backend/services/organization-service/src/main/java/com/lmspilot/organization/OrganizationServiceApplication.java
package com.lmspilot.organization;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class OrganizationServiceApplication {
    public static void main(String[] args) { SpringApplication.run(OrganizationServiceApplication.class, args); }
}

File: backend/services/organization-service/src/main/kotlin/com/lmspilot/organization/OrganizationServiceApplication.kt
package com.lmspilot.organization

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class OrganizationServiceApplication

fun main(args: Array<String>) {
    runApplication<OrganizationServiceApplication>(*args)
}
